### PR TITLE
Ab automation

### PR DESCRIPTION
**Pull request by acbuyan**
made changes to the repository to make sure all list changes we can edit are automated.

### CODE REVIEW

**Comment by rosemaryjoconnor** [comment]
Hi Amanda - it's looking great. What a lot of work! 
I've spotted one thing in **vocab.py statuses_rename**, **Endangered** is being set twice so it will be set to **Vulnerable**:

   "Victoria": {
          "Extinct": "Extinct",
          "Extinct in the Wild": "Extinct in the Wild",
          "Endangered (Extinct in Victoria)": "Extinct in the Wild",
          "Critically Endangered": "Critically Endangered",
          **"Endangered": "Endangered",
          "Endangered": "Vulnerable",**
          "Conservation Dependent": "Conservation Dependent",
          "Restricted": "Restricted"
      },

I only spotted 2 very minor typos in comments:

- **ingest_lists.py**  Line 18: 'states' should be 'state'
- **list_functions.py** Line 63: 'colums' should be 'columns'  (I'm thinking this may have been mine originally!)

A couple of minor questions:

- **vocab.py**: for the various dictionaries is there a process going forward to keep these updated. Particularly the species corrections. 
- Email addresses - will there be a generic email address that goes to a group of recipients

File: functions/sensitive_vs_conservation.py
Comment (Sherrin-ALA) [inline functions/sensitive_vs_conservation.py:1]: I'm a little concerned with the hard coded scientific names being in the code - e.g around lines 51, 94, and 169. Just wondering why they're there. 
Also a little confused by line 105 - is the list coming from Victoria storing taxon rank in the vernacular name column?

**Review by Sherrin-ALA — COMMENTED** [review]
Have really just looked at the sensitive_vs_conservation.py file in detail. Have added some comments there.

**Comment by acbuyan** [comment]
@rosemaryjoconnor : 

Thanks for the review!  I've made the changes you have suggested, and will push them later today.  To answer your questions:

"vocab.py: for the various dictionaries is there a process going forward to keep these updated. Particularly the species corrections."

Not explicitly, no.  From what I remember, these were put in place to make sure that the species matched to something while some of the taxonomy is updated, as well as namematching and sensitive lists.  @Sherrin-ALA can you confirm this?

"Email addresses - will there be a generic email address that goes to a group of recipients"

Yes - "authoritative-list-updates@ala.org.au" - this will email myself, Peggy and Cam (and maybe Simon?)

@Sherrin-ALA:

Looking at your questions, the name added at line 51 is from one of the Jupyter notebooks I was given as the basis of this automation.  I believe that this was added to the list, as it should have been there but wasn't originally on the list.

Line 105 is a typo, and should have been 'TAXON_LEVEL_CDE' - thanks for catching it!  I've edited it and will include it in the latest changes

**Pull request closed without merging**